.travis.yml

```diff
@@ -0,0 +1,13 @@
+sudo: required
+language: java
+services:
+  - docker
+  - postgresql
+  - mysql
+before_install:
+  - ./build/setup_postgres.sh
+  - ./build/setup_mysql.sh
+  - ./build/setup_oracle.sh
+cache:
+  directories:
+    - $HOME/.m2
```

Comment:
GitHub recommends to have empty line at the end of file. Please check all files beyond this with the red mark.